htdocs/core/modules/mrp/doc/pdf_vinci.modules.php

```diff
@@ -1334,9 +1335,15 @@ public function defineColumnField($object, $outputlangs, $hidedetails = 0, $hide
 			'status' => true,
 			'width' => 35, // in mm
 			'title' => array(
-				'textkey' => 'Ref'
+				'textkey' => 'Ref',
+				'align' => 'L',
```

Comment:
Do you really need align => L ? (It should be default value, and tobe compatible with Right to Left languages, having no direction forced is better).

Reply:
The default value is C for titles and R for values, and I suggest in code the same align and padding as used in description

 ```
		// Default field style for content
		$this->defaultContentsFieldsStyle = array(
			'align' => 'R', // R,C,L
			'padding' => array(1, 0.5, 1, 0.5), // Like css 0 => top , 1 => right, 2 => bottom, 3 => left
		);
		// Default field style for content
		$this->defaultTitlesFieldsStyle = array(
			'align' => 'C', // R,C,L
			'padding' => array(0.5, 0, 0.5, 0), // Like css 0 => top , 1 => right, 2 => bottom, 3 => left
		);
```